CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Contributing to napari-gui
+
+We welcome your contributions! Please see the provided steps below and never hesitate to contact us.
+
+If you are a new user, we recommend checking out the detailed [Github Guides](https://guides.github.com).
+
+## Setting up a development installation
+
+In order to make changes to `napari-gui`, you will need to [fork](https://guides.github.com/activities/forking/#fork) the
+[repository](https://github.com/Napari/napari-gui).
+
+If you are not familiar with `git`, we recommend reading up on [this guide](https://guides.github.com/introduction/git-handbook/#basic-git).
+
+Clone the forked repository to your local machine and change directories:
+```sh
+$ git clone https://github.com/your-username/napari-gui.git
+$ cd napari-gui
+```
+
+Set the `upstream` remote to the base `napari-gui` repository:
+```sh
+$ git add upstream https://github.com/Napari/napari-gui.git
+```
+
+Install the required dependencies:
+```sh
+$ pip install -r requirements.txt
+```
+
+Make the development version available globally:
+```sh
+$ pip install -e .
+```
+
+## Making changes
+
+Create a new feature branch:
+```sh
+$ git checkout master -b your-branch-name
+```
+
+`git` will automatically detect changes to a repository.
+You can view them with:
+```sh
+$ git status
+```
+
+Add and commit your changed files:
+```sh
+$ git add my-file-or-directory
+$ git commit -m "my message"
+```
+
+> ### Help us make sure it's you
```

Comment:
Should this really be indented like a quote?

Reply:
TBH I was envisioning this as a “useful aside.” Not sure if a block quote properly captures this or something else would work better.

Reply:
I'm happy unindenting this, but I'm going to approve the PR in the meantime, good stuff. =)